scene/resources/world_3d.h

```diff
@@ -44,6 +44,13 @@ struct SpatialIndexer;
 class World3D : public Resource {
 	GDCLASS(World3D, Resource);
 
+public:
+	enum OcclusionCullingBuildQuality {
+		OCCLUSION_CULLING_BUILD_QUALITY_LOW = 0,
```

Comment:
I also would make this a global setting that you can only access via RenderingServer. Not much of a point to make per viewport, and since you will most likely never change (and keep quality to high).